proto/monitor_service.proto

```diff
@@ -94,12 +94,18 @@ message TieredCacheTracingRequest {
 message TieredCacheTracingResponse {}
 
 message GetProfileStatsRequest {
+  // Executors to fetch statistics for.
   repeated uint64 executor_ids = 1;
+  // Dispatchers do not have executors.
+  // We have to fetch their statistics separately.
+  repeated uint32 dispatcher_fragment_ids = 2;
 }
 
 message GetProfileStatsResponse {
   map<uint64, uint64> stream_node_output_row_count = 1;
   map<uint64, uint64> stream_node_output_blocking_duration_ms = 2;
+  map<uint32, uint64> dispatch_fragment_output_row_count = 3;
+  map<uint32, uint64> dispatch_fragment_output_blocking_duration_ns = 4;
```

Comment:
Is it possible to unify the stream node and dispatch map?

Reply:
I think we should distinguish them, because their processing is different.
Dispatcher metrics are already pre-aggregated, whereas stream node metrics are not.